recipes/pydstool/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.90.2" %}
+{% set sha256 = "6e0cbceab8088ee8cbd0839202cdbaeebbe4ab2e6d1a0d0053c7de30268587c1" %}
+package:
```

Comment:
Can you add a space here? (It can be in the feedstock, this does not hold off merging.)